### PR TITLE
don't error if we can't find a trashbin item for a file when looking …

### PR DESCRIPTION
**Pull request by icewind1991**
…for relevant acl paths

### CODE REVIEW

**Comment by icewind1991** [comment]
Wasn't aware of that issue, butt should resolve it yes.

Haven't locally reproduced the issue

File: lib/ACL/ACLManager.php [hunk 1]
```diff
@@ -116,9 +116,11 @@ private function getRelevantPaths(string $path): array {
 			if ($fromTrashbin && ($path === '__groupfolders/trash')) {
 				/* We are in trash and hit the root folder, continue looking for ACLs on parent folders in original location */
 				$trashItemRow = $this->trashManager->getTrashItemByFileName($groupFolderId, $rootTrashedItemName, $rootTrashedItemDate);
-				$path = dirname('__groupfolders/' . $groupFolderId . '/' . $trashItemRow['original_location']);
 				$fromTrashbin = false;
-				continue;
+				if ($trashItemRow) {
```
Comment (come-nc) [inline lib/ACL/ACLManager.php:120]: We should log something if the file is not found, sounds fishy

**Review by come-nc — APPROVED** [review]

**Comment by solracsf** [comment]
Can we backport this?
Log is flooded on v28 with https://github.com/nextcloud/groupfolders/issues/2794

**Comment by come-nc** [comment]
@icewind1991 You merged with red CI and tests are not compatible :cry:

**Comment by come-nc** [comment]
> @icewind1991 You merged with red CI and tests are not compatible 😢

Already fixed at https://github.com/nextcloud/groupfolders/pull/3043

**Pull request merged**